src/hotspot/os/linux/crac_linux.cpp

```diff
@@ -462,3 +466,46 @@ bool crac::read_bootid(char *dest) {
   }
   return true;
 }
+
+void crac::set_terminate_with_parent() {
+  if (prctl(PR_SET_PDEATHSIG, SIGTERM)) {
```

Comment:
The signal that has killed the parent should be SIGKILL, so it should be specified here as well.